docs/backends/amazon-S3.rst

```diff
@@ -104,6 +104,9 @@ searches for them:
 ``AWS_S3_ENDPOINT_URL`` (optional: default is ``None``)
     Custom S3 URL to use when connecting to S3, including scheme. Overrides ``AWS_S3_REGION_NAME`` and ``AWS_S3_USE_SSL``. To avoid ``AuthorizationQueryParametersError`` error, ``AWS_S3_REGION_NAME`` should also be set.
 
+``AWS_S3_USE_S3_URI_SCHEME`` (optional: default is ``False``)
+    Custom `s3` URI scheme use when connecting to S3.
```

Comment:
Can you change this to `Use an 's3' URI scheme for connections.`

There's nothing really custom about it.

Reply:
Also, this only matters if custom domain is enabled, so that should be mentioned as well.